cpmonitor/tests/cleanupimages_test.py

```diff
@@ -0,0 +1,128 @@
+import os
+import shutil
+import tempfile
+from io import StringIO
+from unittest.mock import patch
+
+from django.core.management import call_command
+from django.test import TestCase
+
+from cpmonitor.models import City
+
+
+class CleanupImagesTest(TestCase):
+    base_path = None
+    images_path = None
+    uploads_path = None
+
+    @classmethod
+    def setUpClass(cls):
+        cls.base_path = tempfile.mkdtemp()
```

Comment:
I don't like that running the test will add files to the /tmp directory of my computer. 
I just know the Java way of doing things which would be to have a test resources folder inside the tests folder where things can be stored temporarily. I actually would just hardcode that and not use a random name.  
An advantage of this is also that the generated files become visible inside the IDE and version control if things go wrong.

Reply:
Now the images will be saved in `tests/test-images/` and the directory is ignored by git